### ISSUE TITLE
Combine all flags into sprite

### ISSUE THREAD

**Issue by slaFFik** [WPGlobus/WPGlobus#1]
And users will be able to use flags in a font-awesome style: 
`<span class="wpglobus-flags wpglobus-flags-ua"></span>`

Or something similar.

**Comment by tivnet** [comment]
@slaFFik Yes, of course. Just need to think how to add a custom flag, if those in the sprite are not enough?
Thank you!

**Comment by slaFFik** [comment]
Just to the end of a file (vertically of horizontally). So all previous coordinates won't break, just new one added.

**Comment by tivnet** [comment]
@slaFFik I mean, if the end user wants to add a flag.

**Comment by slaFFik** [comment]
Something like this (more polished and escaped) can do the trick:

```php
function wpg_get_flag_html($country_code){

    $html = "<span class='wpglobus-flags wpglobus-flags-{$country_code}'></span>";

    return apply_filters( 'wpg_get_flag_html', $html, $country_code );
}
```

**Comment by tivnet** [comment]
Yes, that would work. We were thinking about a non-programmers interface: upload a flag icon, and select it for a language. But having it both ways, and CSS sprites, won't harm, for sure.
Thanks again!
